modules/op-installing-results.adoc

```diff
@@ -0,0 +1,94 @@
+// This module is included in the following assembly:
+//
+// * cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="installing-results_{context}"]
+= Installing {tekton-results}
+
+[role="_abstract"]
+To install {tekton-results}, you must provide the required resources and then create and apply a `TektonResult` custom resource (CR). The {pipelines-shortname} Operator installs the Results services when you apply the `TektonResult` custom resource.
+
+.Prerequisites
+
+* You installed {pipelines-shortname} using the Operator.
+* You prepared a secret with the SSL certificate.
+* You prepared storage for the logging information.
+* You prepared a secret with the database credentials.
+
+.Procedure
+
+. Create the resource definition file named `result.yaml` based on the following example. You can adjust the settings as necessary.
++
+[source,yaml]
+----
+  apiVersion: operator.tekton.dev/v1alpha1
+  kind: TektonResult
+  metadata:
+    name: result
+  spec:
+    targetNamespace: openshift-pipelines
+    logs_api: true
+    log_level: debug
+    db_port: 5432
+    db_host: tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
+    logs_path: /logs
+    logs_type: File
+    logs_buffer_size: 32768
+    auth_disable: true
+    tls_hostname_override: tekton-results-api-service.openshift-pipelines.svc.cluster.local
+    db_enable_auto_migration: true
+    server_port: 8080
+    prometheus_port: 9090
+----
+
+. Add configuration for the storage for logging information to this file:
+** If you configured a persistent volume claim (PVC), add the following line to provide the name of the PVC:
++
+[source,yaml]
+----
+    logging_pvc_name: tekton-logs
+----
```

Comment:
Should add the following in this and the following code blocks, per the [Doc Guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields):

```
apiVersion: operator.tekton.dev/v1alpha1
kind: TektonResult
metadata:
  name: result
spec:
# ...
```

Reply:
@mburke5678 in this part of the procedure, I do not provide a YAML example, to which the guidance applies, Instead, I am explicitly instructing the user to _add_ lines to a file that the user just created in the previous step. If I put in the full header, the user could possibly add this header again, and this breaks the file.